README.md

```diff
@@ -53,20 +53,25 @@ val ipAddr = "127.0.0.1" // WRONG!
 
 ### View IDs
 
-View ID names (`android:id` in XML) should be all lowercase with underscores separating words.
+View ID names (`android:id` in XML) should be all lowercase with underscores separating words (snake_case).  Do your best to create a name that contains an underscore so it is distinguishable from local variables. 
 
 ```xml
 <android.support.v7.widget.RecyclerView
     android:id="@+id/dance_moves"
     />
+
+<Button
+    android:id="@+id/login"
+    />
 ```
 
 When using [Kotlin Android Extensions] to provide synthetic view properties, use the `as` keyword on [imports] to name
-properties according to [non-constant names Android Kotlin codestyle]. The name should be [camel case], postfixed with
-it's view type:
+properties that do not have an underscore in their name, and append `_view` to the end
+
 
 ```kotlin
-import kotlinx.android.synthetic.main.scan.dance_moves as danceMovesRecyclerView
+import kotlinx.android.synthetic.main.scan.dance_moves
+import kotlinx.android.synthetic.main.scan.login as login_view
```

Comment:
I think one of the benefits of going with the XML name is that we no longer have to go through the trouble of specifying `as` for the `import`.

So perhaps the requirement for having an `_` should be for the XML `id`?